zuchini-model/src/main/java/org/zuchini/parser/FeatureParser.java

```diff
@@ -69,13 +69,15 @@ static Feature parseWithWalker(ANTLRInputStream inputStream) {
     }
```

Comment:
Is `parseWithWalker` used?

Reply:
No :)

I kept it because the project actually started as me trying out writing parsers with antl4 and still could not decide which approach I like better.